projects/igniteui-angular-elements/src/app/app.module.ts

```diff
@@ -31,20 +31,22 @@ import { registerConfig } from "../analyzer/elements.config";
 import { createIgxCustomElement } from './create-custom-element';
 import { IgxGridStateComponent } from '../lib/state.component';
 import { ELEMENTS_TOKEN } from 'igniteui-angular/src/lib/core/utils';
+import { IgxIconBroadcastService } from '../lib/icon.broadcast.service';
 
 @NgModule({
   imports: [
     BrowserModule,
     BrowserAnimationsModule
   ],
   providers: [
-    { provide: ELEMENTS_TOKEN, useValue: true }
+    { provide: ELEMENTS_TOKEN, useValue: true },
+    IgxIconBroadcastService
   ],
 //   bootstrap: []
 })
 export class AppModule {
 
-  constructor(private injector: Injector) {}
+  constructor(private injector: Injector, private _iconBroadcast: IgxIconBroadcastService) {}
```

Comment:
Okay, kind of better we're keeping this in Elements for now.
A bit awkward in terms of usability as it stands having to inject just to trigger the functionality. So it either doesn't need to go through the DI system or perhaps we should consider some API on it like `start` & `stop` basically - both to make sense as usage and to have a cleanup mechanism.
Alternatively, since this isn't root-provided it should be able to take advantage of `ngOnDestroy` for cleanup.
Both of those things technically don't matter much for Elements, since this module will never destroy until the entre app does, but still should be noted.